src/containers/EmailConfirmationContainer.js

```diff
@@ -14,4 +15,8 @@ const EmailConfirmationContainer = React.createClass({
   }
 });
 
-export default EmailConfirmationContainer;
+export const mapStateToProps = ({ confirmation }) => ({ confirmation });
```

Comment:
Your `chooseTranslation` logic needs to go here, based on the result of `confirmation`.

Always create props in container components and then pass them to "dumb" presentational components e.g. `components/EmailConfirmation` for rendering. rather than adding logic inside.